lib/node_modules/@stdlib/utils/homedir/README.md

```diff
@@ -0,0 +1,88 @@
+# homedir
+
+> Return the current user's home directory.
+
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var homedir = require( '@stdlib/lib/node_modules/@stdlib/utils/homedir' );
+```
+
+#### homedir()
+
+Returns the current user's `home` directory.
+
+``` javascript
+var home = homedir();
+// e.g., returns '/Users/<username>'
+```
+
+If unable to locate a `home` directory, the module returns `null`.
+
+``` javascript
```

Comment:
I am not sure we should include this example. Could cause issues if we ever perform automated testing of doc examples.